server/flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Airborne server";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    rust-flake.url = "github:juspay/rust-flake";
+  };
+
+  outputs = inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      imports = [
+        inputs.rust-flake.flakeModules.default
+        inputs.rust-flake.flakeModules.nixpkgs
+        # FIX ME: Import rust.nix and do not update devShells
+        # ./rust.nix
```

Comment:
No need to rust.nix if you have crate.nix. rust-flake auto imports crate.nix in appropriate context.